src/components/navigation/CustomNavigation/CustomNavigation.tsx

```diff
@@ -0,0 +1,125 @@
+import React, {memo, useCallback, useEffect, useState} from 'react';
+import {
+    Col,
+    DesktopNavigation,
+    Grid,
+    NavigationComponentProps,
+    // NavigationItemModel,
+    Row,
+    useActiveNavItem,
+    useShowBorder,
+} from '@gravity-ui/page-constructor';
+import block from 'bem-cn-lite';
+
+import {MobileControlsProps} from '../../MobileControls/MobileControls';
+import Sidebar from '../Sidebar/Sidebar';
+import SidebarContent, {NavigationTocProps} from '../SidebarContent/SidebarContent';
+import {useItemsWithCustomDropdowns} from '../hooks';
+
+import './CustomNavigation.scss';
+
+const CLASS_NAME = 'pc-navigation';
```

Comment:
we reuse navigation styles here or make our own custom one ?

Reply:
Most of our styles, and some of the page-constructor styles, use the .pc-navigation class. Therefore, to avoid problems and just repeat the styles in the previous form, I used this class name